main.go

```diff
@@ -104,19 +103,18 @@ func main() {
 	}
 	go func() {
 		logger.Debug("Listening for reports ...")
+		// Print Report Lines
```

Comment:
I think we don't need this comment. If you think we do please change it to 'Print report lines.`